libtenzir/builtins/operators/delay.cpp

```diff
@@ -137,12 +138,36 @@ class delay_operator final : public crtp_operator<delay_operator> {
   std::optional<time> start_ = {};
 };
 
-class plugin final : public virtual operator_plugin<delay_operator> {
+class plugin final : public virtual operator_plugin<delay_operator>,
+                     public virtual operator_factory_plugin {
 public:
   auto signature() const -> operator_signature override {
     return {.source = true};
   }
 
+  auto make(invocation inv, session ctx) const
+    -> failure_or<operator_ptr> override {
+    // XXX: Perhaps update delay_operator to deal with located<> instead if the
+    // older parser supports it?
```

Comment:
It does now, it didn't at the time when I wrote this operator. So yeah, good idea.